integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/plan/KafkaWriterVisitor.java

```diff
@@ -0,0 +1,36 @@
+package io.openlineage.spark.agent.lifecycle.plan;
+
+import io.openlineage.client.OpenLineage;
+import io.openlineage.spark.agent.util.PlanUtils;
+import io.openlineage.spark.agent.util.ScalaConversionUtils;
+import java.util.Collections;
+import java.util.List;
+import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan;
+import org.apache.spark.sql.execution.datasources.SaveIntoDataSourceCommand;
+import org.apache.spark.sql.kafka010.KafkaSourceProvider;
+
+public class KafkaWriterVisitor
+    extends QueryPlanVisitor<SaveIntoDataSourceCommand, OpenLineage.Dataset> {
+
+  @Override
+  public boolean isDefinedAt(LogicalPlan x) {
+    return x instanceof SaveIntoDataSourceCommand
+        && ((SaveIntoDataSourceCommand) x).dataSource() instanceof KafkaSourceProvider;
+  }
+
+  @Override
+  public List<OpenLineage.Dataset> apply(LogicalPlan x) {
+    SaveIntoDataSourceCommand saveIntoDataSourceCommand = (SaveIntoDataSourceCommand) x;
+
+    String servers =
+        ScalaConversionUtils.asJavaOptional(
+                saveIntoDataSourceCommand.options().get("kafka.bootstrap.servers"))
```

Comment:
servers are comma separated list of hosts
` .option("kafka.bootstrap.servers", "host1:port1,host2:port2")`